vcs/gogpm_test.go

```diff
@@ -90,7 +91,7 @@ func withDummyBuildContextMultipleGopath(t *testing.T, testFunc func(string, str
 	if err != nil {
 		t.Fatal(err)
 	}
-	if err = os.MkdirAll(path.Join(fakeGoPathTwo, "src/github.com/fake/library"), 0777); err != nil {
+	if err = os.MkdirAll(path.Join(fakeGoPathTwo, "src", "github.com", "fake", "library"), 0777); err != nil {
```

Comment:
In this case, it's quite clear you should be using `filepath`, since you're using its result for `os.MkdirAll` and `MkdirAll` expects a system file path.